packages/code-snippet-block/src/CodeSnippetBlock.spec.ct.tsx

```diff
@@ -80,9 +80,9 @@ it('renders code snippet with line numbers', () => {
 it('renders code snippet with border', () => {
     const [CodeSnippetWithStubs] = withAppBridgeBlockStubs(CodeSnippetBlock, {
         blockSettings: {
-            withBorder: true,
-            lineWidth: '2px',
-            lineStyle: 'solid',
+            hasBorder: true,
+            borderWidth: '2px',
+            borderStyle: 'solid',
```

Comment:
these 2 props should also be changes in the next 2 tests  below :)

Reply:
Or they could even be removed in the other tests, because the border setting isn't event tested in those tests 😅